packages/did-core-test-server/suites/methods/did-key/defaultSuiteConfig.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "did-spec",
+  "didMethods": {
+    "did:key": {
+      "supportedContentTypes": [
+        "application/did+json",
+        "application/did+ld+json"
+      ],
+      "dids": ["did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"],
+      "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB": {
+        "application/did+json": {
+          "didDocument": {
+            "@context": [
+              "https://www.w3.org/ns/did/v1",
+              {
+                "@base": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+              }
+            ],
+            "id": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+            "verificationMethod": [
+              {
+                "id": "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+                "type": "JsonWebKey2020",
+                "controller": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+                "publicKeyJwk": {
+                  "crv": "Ed25519",
+                  "x": "LKacPea7et-QMhcyS02ZjsTHEVadsCj_XNnfFJYbwHo",
+                  "kty": "OKP"
+                }
+              },
+              {
+                "id": "#z6LSh8yKFPF2bLnD9mHQZMJL6zQQZsowYUeUaDd39rMLHBBx",
+                "type": "JsonWebKey2020",
+                "controller": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+                "publicKeyJwk": {
+                  "kty": "OKP",
+                  "crv": "X25519",
+                  "x": "USe317eZOgxr3-neAHaJ8fZ056kb7miDxZvb5ymFKVM"
+                }
+              }
+            ],
+            "authentication": [
+              "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+            ],
+            "assertionMethod": [
+              "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+            ],
+            "capabilityInvocation": [
+              "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+            ],
+            "capabilityDelegation": [
+              "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+            ],
+            "keyAgreement": [
+              "#z6LSh8yKFPF2bLnD9mHQZMJL6zQQZsowYUeUaDd39rMLHBBx"
+            ]
+          },
+          "didDocumentMetaData": {
+            "content-type": "application/did+json"
+          },
+          "didResolutionMetaData": {}
+        },
+        "application/did+ld+json": {
+          "didDocument": {
+            "@context": [
+              "https://www.w3.org/ns/did/v1",
+              {
+                "@base": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+              }
+            ],
+            "id": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+            "verificationMethod": [
+              {
+                "id": "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
+                "type": "Ed25519VerificationKey2018",
```

Comment:
Just as a heads-up, we're probably going to submit a `Ed25519VerificationKey2020` for did:key with `publicKeyMultibase` -- I assume the test suite allows for multiple did:key submissions?